celery/app/task.py

```diff
@@ -92,6 +92,7 @@ class Context(object):
     errbacks = None
     timelimit = None
     origin = None
+    task = None
```

Comment:
@thedrow Thanks for looking!   I did update the user guide, see:
https://github.com/celery/celery/pull/4490/files#diff-cc90971f3ec1a23279b1aa3e1426d247

Since it's internal, I'll change it to task_name.